File: Problem 0001/Python 3/problem_1.py
def multiples_of(number_1: int, number_2: int, below_range: int):
    sum = 0
    multiplier = 1

    while (multiplier <= below_range):
        if (number_1 * multiplier) < below_range:
            sum += number_1 * multiplier
        if (number_2 * multiplier) < below_range:
            sum += number_2 * multiplier
        multiplier += 1
    print(sum)

if __name__ == '__main__':
    multiples_of(3, 5, 1000)

File: Problem 0001/Java/Problem1.java
class Problem1 {

  static int multiplesOf(int number1, int number2, int belowRange) {
    int sum = 0;
    int multiplier = 1;

    while (multiplier <= belowRange) {
      if ((number1 * multiplier) < belowRange) {
        sum += number1 * multiplier;
      }
      if ((number2 * multiplier) < belowRange) {
        sum += number2 * multiplier;
      }
      multiplier += 1;
    }
    return sum;
  }

  public static void main(String[] args) {
    var test1 = multiplesOf(3, 5, 10);
    var test2 = multiplesOf(3, 5, 1000);
    System.out.println(test1);
    System.out.println(test2);
  }
}
